requirements.txt

```
@@ -1,33 +1,33 @@
-alembic==1.5.4
+alembic==1.13.1
 attrs==20.3.0
 autopep8==1.5.5
-blinker==1.4
+blinker==1.7
 certifi==2020.12.5
 chardet==4.0.0
-click==7.1.2
-Flask==1.1.2
-Flask-Migrate==2.6.0
-Flask-SQLAlchemy==2.4.4
+click==8.1.7
+Flask==3.0.2
+Flask-Migrate==4.0.5
+Flask-SQLAlchemy==3.1.1
 gunicorn==20.1.0
 idna==2.10
 iniconfig==1.1.1
-itsdangerous==1.1.0
-Jinja2==2.11.3
+itsdangerous==2.1.2
+Jinja2==3.1.3
 Mako==1.1.4
-MarkupSafe==1.1.1
+MarkupSafe==2.1.5
 packaging==20.9
-pluggy==0.13.1
+pluggy==1.4.0
 psycopg2-binary==2.9.5
```

I got a slightly newer psycopg2 version when I was building a test app
```suggestion
psycopg2-binary==2.9.9
```